idunn/api/geocoder.py

```diff
@@ -11,7 +11,12 @@
     async def get_intentions():
         if not query.nlu:
             return None
-        return await nlu_client.get_intentions(text=query.q, lang=query.lang)
+
+        focus = None
+        if query.lon and query.lat:
+            focus = (query.lat, query.lon)
```

Comment:
If I'm not wrong, these values are used as (x,y) to initialize a shapely `Point`, so they should be reversed.  
Maybe it would be clearer to create a `Point` instance here.

Reply:
Grrrrr, I still manage to get confused with this even when I try to look for it ... :angry:

Reply:
It should be fixed in 119d1a4  though